wormholes:
  - map_a: room_a
    map_b: room_b
    region_a: "POLYGON((1 0,1 2,2 2,2 0,1 0))"
    region_b: "POLYGON((0 1,0 3,1 3,1 1,0 1))"
